signin/signin.js

```diff
@@ -0,0 +1,65 @@
+const email = document.querySelector("#email");
+const password = document.querySelector("#password");
+const emailError = document.querySelector(".emailError");
+const passwordError = document.querySelector(".passwordError");
+
+const emailRegex = new RegExp("[a-z0-9]+@[a-z]+.[a-z]{2,3}");
+
+const form = document.querySelector("#form");
+
+const eyeOpen = document.querySelector(".eyeOpen");
+const eyeClose = document.querySelector(".eyeClose");
+
+function onSubmit(event) {
+  event.preventDefault();
+
+  if (email.value === "test@codeit.com" && password.value === "codeit101") {
```

Comment:
보통 이런 경우 test 계정을 상수로 선언해두고 여기서는 가져다 쓰면 실수를 미연에 방지할 수 있습니다.
const TEST_EMAIL = 'test@codeit.com'
const TEST_PW = 'codeit101'

```suggestion
  if (email.value === TEST_EMAIL && password.value === TEST_PW) {
```